packages/next/src/lib/typescript/writeConfigurationDefaults.ts

```diff
@@ -374,9 +334,9 @@ export async function writeConfigurationDefaults(
   }
 
   if (!('exclude' in userTsConfig)) {
-    userTsConfig.exclude = ['node_modules']
+    userTsConfig.exclude = ['node_modules', '.next']
```

Comment:
We don't actually need to include `.next` since dot folders are excluded by default in TypeScript. Please don't add `.next` since it disables route and link type-checking in existing apps.